wordle/readme.md

```diff
@@ -0,0 +1,33 @@
+# Wordle CLI Workshop
+
+This is a workshop where we, as a class, build a Wordle CLI clone in Node JS.
+
+## Learning Objectives
+
+```objectives
+- [ ] Give trainees the experience of building a fully playable game from scratch, starting with nothing but an idea
+- [ ] Give trainees some understanding of how to find and use NPM dependencies in a project
+- [ ] Walk trainees through the process of breaking down a high-level idea into smaller, less intimidating bits of functionality
+- [ ] Inspire trainees to take their own ideas and build something cool with whatever skills they have
+```
+
+## Requirements
+
+- Split the cohort into groups of 3 to 4 people
+- Introduction slides: https://barath-cyf.netlify.app/wordle/
+- We start with an energiser where we play an actual game of Wordle (https://www.nytimes.com/games/wordle/index.html) - sourcing guesses from the class, and hopefully learning how the game works in the process
+- Ensure the class understands the rules of the game first
+- Then we start building out our Wordle CLI clone from scratch
+    - Referencing steps from this README: https://github.com/CodeYourFuture/wordle-clone-example#readme (this example solution can be given to the trainees after the session has finished)
+    - Steps 1 & 2 can be done together as a class. All the code here should be sourced from the trainees themselves - this is their implementation. However, the volunteer who is running the workshop can ensure the high-level problem is broken down into these smaller functions: getRandomWord, getGuess and showWordWithHighlights
+    - Step 3 (getRandomWord) can be done in groups
+    - Step 4 (getGuess) can be done in groups - but it might be useful for a volunteer to suggest which NPM dependency we could use
+    - Step 5 (showWordWithHighlights) can be done in groups - again, we can suggest which NPM dependency to use
+    - Step 6 (improving the user experience) could possibly be done as a class together
+- Back to the slides (https://barath-cyf.netlify.app/wordle/), we can wrap up the session by suggesting some stretch ideas, and again emphasizing the power of trainee ideas
+
+## Acceptance Criteria
+
+- I have a working Wordle CLI game which I can play
```

Comment:
```suggestion
_Given_ the user starts a new Wordle game
_When_ the app is initialized
_Then_ a random 5 letter word should be selected as the target word

_Given_ the target word has been selected
_When_ the user enters a 5 letter guess
_Then_ the guess should be validated
_And_ each letter should be marked as green, yellow or gray based on whether it matches the target word

_Given_ a valid 5 letter guess has been entered
_When_ the guess does not match the target word
_Then_ each letter should be marked as follows:
- Green if the letter is in the correct position
- Yellow if the letter is in the target but not in the correct position
- Gray if the letter is not in the target word

_Given_ a guess has been submitted
_When_ the guess matches the target word
_Then_ a message should indicate the user has won the game

_Given_ guesses have been submitted
_When_ the user reaches 6 guesses without matching the target word
_Then_ a message should indicate the user has lost and show the target word
```

Reply:
@moneyinthesky A lot of the curriculum has moved towards more formal Given/When/Then style acceptance criteria and it would be a good opportunity to use them here. I just generated these using Claude so feel free to tweak them as necessary - I also suggest hoisting these criteria further up in the readme